Dockerfile

```diff
@@ -25,8 +25,19 @@ RUN apt-get install -y apt-utils \
   && apt-get install -y --no-install-recommends curl libxml2-dev libxslt1-dev g++ gcc git gnupg2 make openssh-client ruby-dev wget zlib1g-dev \
   && wget https://apt.puppet.com/puppet-tools-release-buster.deb \
   && dpkg -i puppet-tools-release-buster.deb \
-  && apt-get update -qq \
-  && apt-get install -y --no-install-recommends pdk \
+  && apt-get update -qq
+
+# install old PDK and run tests to pull in gem dependencies
+RUN apt-get install -y --no-install-recommends pdk=1.18.1.0-1buster \
+  && yes | pdk new module test --skip-interview \
+  && cd test \
+  && pdk validate \
+  && pdk test unit \
```

Comment:
I have no experience with `pdk` but I'd hope there's commands that will directly/explicitly pull down (prod) dependencies; all software package managers end up with functionality.

Running tests to do this is odd conceptually, but it also has the downside of pulling in *test* dependencies as well. This means we'd be shipping test dependencies in the image, which is probably a deal-breaker considering security concerns?

We should re-visit `pdk` CLI and see if there is a more suitable command to pull down dependencies?
And then whatever is resulting from the `new module` call can/should just be put in place directly rather than going through a mode meant for human interaction. I assume that probably just means a certain file structure?